unbxdSearch.js

```diff
@@ -1226,7 +1226,7 @@ var unbxdSearchInit = function (jQuery, Handlebars) {
             } else if (this.options.type == "browse" && this.params['categoryId'] != undefined) {
                 url += '&category-id=' + encodeURIComponent(this.params.categoryId);
             } else if (isTypeCategory(this.options.type) && this.params.categoryId !== undefined) {
-                url += "&p=" + encodeURIComponent(this.params.categoryId);
+                url += "&p=categoryPath:\"" + encodeURIComponent(this.params.categoryId) + "\"" + '&pagetype=boolean';
```

Comment:
Dont hardcode `categoryPath`, it will not work for other attribute pages.
For eg: on this page: https://www.ecklers.com/camaro
API call will be `p: carline:"camaro"`

Reply:
Done